dapr/proto/scheduler/scheduler_callback.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package dapr.proto.scheduler.v1;
+
+import "google/protobuf/empty.proto";
+
+service SchedulerCallback {
+    // Callback RPC for job schedule being at 'trigger' time
+    rpc TriggerJob(TriggerRequest) returns (google.protobuf.Empty) {}
+}
+
+// TriggerRequest is the message request to convey that the job is at its 'trigger' time. 
+message TriggerRequest {
+    // The job.
+    string job_name = 1;
```

Comment:
I think this will need to be the entire job spec, not just the job name bc we are sending back the job details to the app with the 'triggerJob'. see the request with the entire job spec in [the proposal](https://github.com/dapr/proposals/pull/44/files) line 426

Reply:
Yeah, I thought about it. To me it seemed, that we just need to let sidecar know which job to be triggered. Why do we need to send across the schedule details etc. here? It just needs to be triggered with corresponding metadata details.
Please let me know, if I am missing something here.

Reply:
We are sending back the entire job spec back to the app at the 'trigger' time. That way the app can do whatever logic at that time they would like. They are not expected to do job lookup based on name themselves to figure out what is suppose to occur logic wise for that job. That is why we want to send back the entire job spec.